lib/presentation/register/view/register_page_view.dart

```diff
@@ -0,0 +1,403 @@
+                                    fontSize: 15, color: AppColors.black),
+                              )
+                            ],
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                  title: Text(AppLocalizations.of(context)!.register,
+                      style: AppTextStyles.titleLogin()),
+                  centerTitle: true,
+                  toolbarHeight: 75,
+                  backgroundColor: AppColors.brown.withOpacity(0.8),
+                  elevation: 0,
+                  leading: IconButton(
+                      onPressed: () {
+                        Navigator.maybePop(context);
+                      },
+                      icon: Icon(Icons.arrow_back_ios))),
+            ),
+            body: SingleChildScrollView(
+              keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
+              child: buildBody(),
+            )),
+        onWillPop: () => onBackPressed(context));
+  }
+
+  Widget buildBody() {
+    return Form(
+      key: _formKey,
+      child: Column(
+        children: [
+          SizedBox(
+            height: 20,
+          ),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 40),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                inputForm(),
+                SizedBox(
+                  height: 40,
+                ),
+                Center(
+                  child: Container(
+                    width: 200,
+                    height: 50,
+                    decoration: BoxDecoration(
+                      color: AppColors.white,
+                      borderRadius: BorderRadius.circular(25),
+                    ),
+                    child: Center(
+                      child: SizedBox.expand(
+                        child: ElevatedButton(
+                            style: ElevatedButton.styleFrom(
+                                backgroundColor:
+                                    AppColors.brown.withOpacity(0.8)),
+                            onPressed: () {
+                              onSignUpPressed();
+                            },
+                            child: Text(
+                              AppLocalizations.of(context)!.register,
+                              style: TextStyle(
+                                  fontWeight: FontWeight.bold,
+                                  color: AppColors.black),
+                            )),
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          )
+        ],
+      ),
+    );
+  }
+
+  Widget inputForm() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(
+          height: 10,
+        ),
+        TextFormField(
```

Comment:
Optional: create a generic widget for your custom TextFormField (you can read here: https://kcl.hashnode.dev/input-form-design-flutter)